packages/package-json-lint/src/lint.js

```diff
@@ -1,7 +1,43 @@
-const { Logger } = require('@cerner/terra-cli');
+const fs = require('fs-extra');
+const rules = require('./rules');
+const PackageIssues = require('./issues/PackageIssues');
+const AggregateIssues = require('./issues/AggregateIssues');
+const { getRuleConfig, getConfig } = require('./config');
+const { getPathsForPackages } = require('./utilities');
 
-const logger = new Logger({ prefix: '[terra-functional-testing:wdio]' });
+const lint = ({ packageJsonData, config }) => {
+  const issues = [];
+  Object.entries(config.rules).forEach(([ruleId, ruleInformation]) => {
+    const rule = rules[ruleId];
+    const ruleConfig = getRuleConfig({ rule, ruleInformation });
+    if (ruleConfig.severity !== 'off') {
+      const issue = rule.lint({ packageJsonData, ruleConfig });
```

Comment:
So how you have this designed now you'll analyze the package data once per rule. Depending on how many rules we add it may be more efficient to read the package data once and apply the rules where appropriate. I think this is how other linters are applied. For the two rules we have now your current approach is probably fine, but something to think about for the future.

Reply:
For example rules could subscribe to sections of the package.json they care about and as we read through the document we could send the info to each rule.

So a "dependency" rule would get called with (dependency, version, type(dep, peer, dev)) and each rule could evaluate and return its results.

Reply:
Yeah I thought about that a little bit. Could make it easier to make a vs code plugin I think. Hmmm.

Reply:
This will be fixed by my big refactoring